guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Here, we use the <code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.gis.toc.html#arcgis.gis.DatastoreManager.search" target="_blank">search()</a></code> method on a <code>DatastoreManager</code> object to search for <code>Datastores</code>. Observe in the output below the item titled <code>ServiceCallsOrleans</code>. As illustrated in the example file structure above, the item is registered as a big data file share in the portal.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>we use the search()</p>